php/php.editor/src/org/netbeans/modules/php/editor/indent/FormatVisitor.java

```diff
@@ -1172,41 +1166,62 @@ public void visit(FunctionDeclaration node) {
 
     @Override
     public void visit(LambdaFunctionDeclaration node) {
```

Comment:
I've rewritten almost of all this implementation because we should consider [psr-12](https://www.php-fig.org/psr/psr-12/). (parameters and lexical variables were not wrapped.)
Formatting problems related to closures are likely to occur in an array/parameters. So especially, I've added unit tests for in arrays.

Existing all unit tests also have passed. I just hope that there is no big problem...

Reply:
Well, we will see :)

If the tests pass, let's wait. It would be fine if there was an easy way to fix possible issues once a new version of NetBeans is released.